src/process/drydep/ccpr_scheme_GOCART_DryDep_mod.F90

```diff
@@ -65,21 +64,19 @@ subroutine CCPr_Scheme_GOCART_DryDep(km,
       REAL, pointer, intent(in), DIMENSION(:) :: tmpu   ! Temperature [K]
       REAL, pointer, intent(in), DIMENSION(:) :: rhoa   ! Air density [kg/m^3]
       REAL, pointer, intent(in), DIMENSION(:) :: hghte  ! Height [m]
-      REAL, pointer, intent(in)                      :: radius                                ! particle radius [m]
-      REAL, pointer, intent(in)                      :: rhop                                  ! particle density [kg/m^3]
+      REAL, pointer, intent(in), optional            :: radius                                ! particle radius [m]
+      REAL, pointer, intent(in), optional            :: rhop                                  ! particle density [kg/m^3]
       REAL, pointer, intent(in)                      :: ustar                                 ! friction speed [m/sec]
       REAL, pointer, intent(in)                      :: pblh                                  ! PBL height [m]
       REAL, pointer, intent(in)                      :: hflux                                 ! sfc. sens. heat flux [W m-2]
       REAL, pointer, intent(in)                      :: z0h                                   ! rough height, sens. heat [m]
-      REAL, pointer, intent(in) :: u10m                   ! 10-m u-wind component [m/sec]
-      REAL, pointer, intent(in) :: v10m                   ! 10-m v-wind component [m/sec]
-      REAL, pointer, intent(in) :: fraclake               ! fraction covered by water [1]
-      REAL, pointer, intent(in) :: gwettop                ! fraction soil moisture [1]
-      real, intent(in) :: radius 
-      real, intent(in) :: rhop
+      REAL, pointer, intent(in), optional :: u10m                   ! 10-m u-wind component [m/sec]
```

Comment:
would make these required to the scheme itself

Reply:
done.